awx/main/analytics/collectors.py

```diff
@@ -254,6 +254,8 @@ def _copy_table(table, query, path):
                               main_jobevent.task,
                               main_jobevent.role, 
                               main_jobevent.job_id, 
+                              main_jobevent.inventory_id, 
```

Comment:
I think you need `main_inventory.id,` here instead.

Reply:
I actually need main_job.inventory_id so that I can match it to the inventory table, should be fixed